rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -29,9 +29,11 @@ namespace rsx
 	
 	std::array<rsx_method_t, 0x10000 / 4> methods{};
 
-	[[noreturn]] void invalid_method(thread*, u32 _reg, u32 arg)
+	void invalid_method(thread*, u32 _reg, u32 arg)
 	{
-		fmt::throw_exception("Invalid RSX method 0x%x (arg=0x%x)" HERE, _reg << 2, arg);
+		//Don't throw, gather information and ignore broken/garbage commands
+		//TODO: Investigate why these commands are executed at all. (Heap corruption? Alignment padding?)
+		LOG_ERROR(RSX, "Invalid RSX method 0x%x (arg=0x%x)" HERE, _reg << 2, arg);
```

Comment:
Thought so as well, but the range of values we're seeing is too large. I'm convinced some are from broken command buffers.
It's difficult to say what happens if you pass broken instructions to a real PS3, I'm guessing they'll be ignored. For some games, the instructions seem to have no effect on visual output (probably undocumented NOPs). We only have one NOP documented, (0x1718 >> 2) and it is handled.